From c4634f5bb446d1c8604a79fee1b3310221c89efd Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 4 Mar 2011 11:31:36 +0100
Subject: [PATCH] plymouth: execute emergency script first

The boot screen has to be shown on emergency first.
---
 modules.d/50plymouth/install | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/modules.d/50plymouth/install b/modules.d/50plymouth/install
index 86d8efd..bac7b6d 100755
--- a/modules.d/50plymouth/install
+++ b/modules.d/50plymouth/install
@@ -11,6 +11,6 @@ fi
 
 inst_hook pre-pivot 90 "$moddir"/plymouth-newroot.sh
 inst_hook pre-trigger 10 "$moddir"/plymouth-pretrigger.sh
-inst_hook emergency 50 "$moddir"/plymouth-emergency.sh
+inst_hook emergency 00 "$moddir"/plymouth-emergency.sh
 inst "$moddir"/cryptroot-ask.sh /sbin/cryptroot-ask
-inst readlink
\ No newline at end of file
+inst readlink
